File: annotationprocessingdemo/src/main/java/com/client/annotationprocessingdemo/MainActivity.java
package com.client.annotationprocessingdemo;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.TextView;

import com.client.lib_reflect_binding.BindView;
import com.client.lib_reflect_binding.Binding;


public class MainActivity extends AppCompatActivity {
    @BindView(R.id.textView)
    TextView textView;
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Binding.bind(this);
        textView.setText("ZhangSan");
    }
}

File: customview/src/main/java/com/client/customview/MainActivity.kt
package com.client.customview

import android.animation.AnimatorSet
import android.animation.ObjectAnimator
import android.animation.PropertyValuesHolder
import android.animation.ValueAnimator
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.view.animation.AccelerateInterpolator
import android.view.animation.AnimationSet
import android.view.animation.LinearInterpolator
import com.client.common.dp
import com.client.customview.widgets.AnimatorView
import com.client.customview.widgets.CanvasView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }


}